Modules/IO/GDCM/test/CMakeLists.txt

```diff
@@ -144,6 +145,15 @@ itk_add_test(NAME itkGDCMLoadImageNoSpacingTest
     1.0
   )
 
+itk_add_test(NAME itkGDCMLegacyMultiFrameTest
+  COMMAND ITKIOGDCMTestDriver
```

Comment:
I think it would be quite helpful to make this more accessible to broader group of users if direct download pointers for the datasets could be added in the comments here. It takes quite a lot of time and space to checkout/configure/download ITK source and data. I also think there should be a link to the original MR series that was converted into the legacy enhanced dataset, since the ultimate test is whether the geometry (and slice order) of the image volume is the same while loading from the original non-enhanced series and the legacy enhanced one.

Reply:
Here's the link to the original non-enhanced MR series:  https://www.dropbox.com/s/8m7ugu4cmw83fvd/dicoms-anon.zip?dl=0